src/vaadin-menu-bar-buttons-mixin.html

```diff
@@ -82,6 +88,21 @@
       }
     }
 
+    __updateButtonsTheme() {
+      Array.from(this.shadowRoot.querySelectorAll('vaadin-menu-bar-button'))
+        .forEach(button => {
+          if (this.theme && this.theme !== '') {
```

Comment:
note: you can move this check outside of `forEach`

Reply:
No big differences as it will require another `forEach` in that case (to add attribute and one for removing it).